ui/src/chat/ChatReactions/ReactionsWidget.tsx

```diff
@@ -0,0 +1,123 @@
+import _ from 'lodash';
```

Comment:
I'm not sure if this has been benchmarked for this repo, but you may want to use `import invertBy from 'lodash/invertBy';` to avoid import bloat: https://www.blazemeter.com/blog/import-lodash-libraries

Reply:
We use this import style for lodash pretty regularly in the repo. From an ergonomic standpoint, it's nice to have them all accessible without needing to think about explicit imports.

We probably should do a more serious pass around import bloat at some point, but that's outside the scope of this update. Ideally, we could keep our current import style and ensure tree shaking is appropriately handled by the compiler.